docs/site/Decorators.md

```diff
@@ -255,12 +242,17 @@ dependencies:
 Syntax: `@inject.getter(bindingKey: string)`.
```

Comment:
link to https://apidocs.strongloop.com/@loopback%2fcontext/#35 to be consistent?